package.json

```diff
@@ -28,9 +28,11 @@
   },
   "dependencies": {
     "chalk": "5.3.0",
+    "cheerio": "^1.0.0-rc.12",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
+    "node-fetch": "^3.3.2",
```

Comment:
Do we not want to rely on the native node [fetch](https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#fetch), which should be available as long as were using a version greater than `18` (stable since version `21`)? I guess we don't set a required min version, so this would ensure it always works, regardless of local version being used.

 I'm not super opinionated on this since this is a testing repo. More wondering if there is a reason.

Reply:
Good catch, will fix.